pkg/apis/serving/v1alpha1/kfservice_framework_tensorrt.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+)
+
+var (
+	TensorRTISImageName             = "nvcr.io/nvidia/tensorrtserver"
+	DefaultTensorRTISRuntimeVersion = "19.05-py3"
+	TensorRTISEntrypointCommand     = "trtserver"
+	InvalidModelURIError            = "Model URI must be prefixed by gs:// (only Google Cloud Storage paths are supported)"
```

Comment:
This probably won't hold after your init container work :)

Reply:
Yes - yay :)